tests/integration/components/github-connect-test.js

```diff
@@ -0,0 +1,42 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import PageObject from 'ember-cli-page-object';
+import component from 'code-corps-ember/tests/pages/components/github-connect';
+
+const { set } = Ember;
+let page = PageObject.create(component);
+
+moduleForComponent('github-connect', 'Integration | Component | github connect', {
+  integration: true,
+  beforeEach() {
+    page.setContext(this);
+  },
+  afterEach() {
+    page.removeContext();
+  }
+});
+
+test('check if scope is bound correctly', function(assert) {
+  set(this, 'scope', '3098379083');
+  this.render(hbs`{{github-connect scope=scope}}`);
```

Comment:
They should not be in a real life use case, but there is little reason to enforce that. 

I would, however, suggest not to write as many tests. A single test which checks various combinations of property values ought to be good enough.